src/ipc/Coordinator.cc

```diff
@@ -78,13 +79,23 @@ void Ipc::Coordinator::registerStrand(const StrandCoord& strand)
 
 void Ipc::Coordinator::receive(const TypedMsgHdr& message)
 {
-    switch (message.type()) {
+    switch (message.rawType()) {
```

Comment:
I think we should try to eliminate these stale/duplicated messages.
This problem is partly handled already because each UDS socket does unlink() before it starts listening - so all messages sent before this event will be lost (or ignored by the new process). The problem may happen when, for example, a kid sends a registration request and dies, and Coordinator responds to the restarted(unaware) worker. We can address this problem similarly as in 30fe12b, i.e., each requestor validates its pid upon receiving response.